spec/04-basic-concepts.md

```diff
@@ -1309,7 +1309,8 @@ three kinds: automatic, static, and allocated.
 
 A variable having *automatic storage duration* comes into being and is
 initialized at its declaration or on its first use, if it has no
-declaration. Its lifetime is delimited by an enclosing scope ([§§](#scope)). The
+declaration (notice that this is simplified; for more details see [§§](07-variables.md#local-variables) for more information).
```

Comment:
It says both "for more details" and "for more information". I suggest it only needs one.